lib/ansible/plugins/callback/default.py

```diff
@@ -152,7 +152,7 @@ def _print_task_banner(self, task):
             args = u' %s' % args
 
         self._display.banner(u"TASK [%s%s]" % (task.get_name().strip(), args))
-        if self._display.verbosity >= 2:
+        if self._display.verbosity >= 2 or task.verbosity >= 2:
```

Comment:
Currently it seems like this is the only place where the task-specific increased verbosity is working as intended.

Reply:
Nevermind, I figured it out.